features/telemetry/daemon/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,56 @@
+<blueprint
```

Comment:
telemetry daemon is not part of Minion so this blueprint won't be loaded on Minion

DistPollerDao also doesn't gets loaded on Minion